From: Remi Collet <remi@remirepo.net>
Date: Mon, 19 Dec 2022 09:24:02 +0100
Subject: NEWS

(cherry picked from commit 7328f3a0344806b846bd05657bdce96e47810bf0)
(cherry picked from commit dbfbd99e91701c0a5613133c06305fd70545e9ad)
---
 NEWS | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/NEWS b/NEWS
index 5e728f7..57a021d 100644
--- a/NEWS
+++ b/NEWS
@@ -1,6 +1,12 @@
 PHP                                                                        NEWS
 |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
 
+Backported from 8.0.27
+
+- PDO/SQLite:
+  . Fixed bug #81740 (PDO::quote() may return unquoted string).
+    (CVE-2022-31631) (cmb)
+
 Backported from 7.4.32
 
 - Core:
